### PR TITLE
luna-sysmgr: Make LaunchAtBoot applications list configurable at the bitbake level

### PR DESCRIPTION
**Pull request by ericblade**
- Add LUNEOS_BOOT_APPS and override the existing .conf line

### CODE REVIEW

File: meta-luneos/recipes-webos-owo/luna-sysmgr/luna-sysmgr.bb [hunk 1]
```diff
@@ -53,6 +56,12 @@ do_install:append() {
         install -v -m 644 ${S}/conf/luna.conf ${D}${webos_sysconfdir}
     fi
 
+    # Apply configured boot applications
+    if [ -f ${D}${webos_sysconfdir}/luna.conf ]
+    then
+        sed -i "1,\$s/Applications=org.webosports.app.phone;com.palm.app.email;com.palm.app.calendar/Applications=${LUNEOS_BOOT_APPS}/" ${D}${webos_sysconfdir}/luna.conf
```
Comment (Tofee) [inline meta-luneos/recipes-webos-owo/luna-sysmgr/luna-sysmgr.bb:62]: Minor remark: I would have put 
`s/Applications=.*/Applications=${LUNEOS_BOOT_APPS}/`
instead, to reflect the fact that we want to override any existing value

**Review by Tofee — APPROVED** [review]
Good idea 👍

**Comment by Tofee** [comment]
One additional remark: as on the other PR, please prefix the commit message with "<recipe>: "

File: meta-luneos/recipes-webos-owo/luna-sysmgr/luna-sysmgr.bb (hunk 1 above)
Comment (ericblade) [reply meta-luneos/recipes-webos-owo/luna-sysmgr/luna-sysmgr.bb:62]: Fair. I'd need to make this match the section, then, so don't take until I figure that out

**Review by Tofee — APPROVED** [review]

**Pull request merged**